app/views/content_items/travel_advice.html.erb

```diff
@@ -16,13 +16,15 @@
   <div class="govuk-grid-column-two-thirds travel-advice__header">
     <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
 
-     <% @content_item.alert_status.each do |text| %>
-      <%= render "govuk_publishing_components/components/warning_text", {
-        text: text % {:country => @content_item.country_name
-      } 
-      } %>
+    <%= render "govuk_publishing_components/components/govspeak", {
+    } do %>
+       <% @content_item.alert_status.each do |text| %>
+        <%= render "govuk_publishing_components/components/warning_text", {
+          text: text % {:country => @content_item.country_name} 
```

Comment:
I've never seen the `%` operator used in this way. Is this supposed to pass the country name to the I18n file? I'm not sure if this is working.

Reply:
I _think_ it was working. But I've changed it to the syntax we usually use 👍